apps/nxls-e2e/src/nxls-wrapper.ts

```diff
@@ -134,6 +137,14 @@ export class NxlsWrapper {
 
     this.process?.removeListener('exit', this.earlyExitListener);
 
+    try {
+      execSync(`npx nx@${version ?? defaultVersion} daemon --stop`, {
```

Comment:
Should there be a windowsHide here as well?

Reply:
sure, not important tho as it's just the e2e